backend/src/vscode-youi-events.ts

```diff
@@ -48,31 +52,45 @@ export class VSCodeYouiEvents implements YouiEvents {
         });
     }
 
-    private showDoneMessage(success: boolean, message: string, targetPath: string): void {
+    private showDoneMessage(success: boolean, errorMmessage: string, targetPath: string): Thenable<any> {
```

Comment:
here it's - success: boolean
at doGeneratorDone it's generationSucceeded: boolean
I prefer success in both

Reply:
Done